### PR TITLE
Merge Authentication to master

### PR DESCRIPTION
**Pull request by ttxking**
* Merge auth-system branch to master to increase code coverage and add most tests for Travis-CI to check.

### CODE REVIEW

**Review by bemyXmas — APPROVED** [review]
Review it and all test is looking good. Great job!

**Comment by ttxking** [comment]
> Review it and all test is looking good. Great job!

Ok. 👍  Thank you for reviewing!

File: accounts/tests/test_register.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    def test_register_with_only_numeric_password(self):
+        """If the password contains only numeric, the error message will be shown."""
+        response = self.client.post(reverse('register'), {
+            'username': 'testuser', 'email': 'testuser2@gmail.com', 'first_name': 'Testerman2', 'password1': '123456',
+            'password2': '123456'})
+        self.assertContains(response, 'This password is entirely numeric.')
+
+    def test_register_with_short_password(self):
+        """If the password is shorter than 8 characters, the error message will be shown."""
+        """If the password contains only numeric, the error message will be shown."""
```
Comment (nicenicegame) [inline accounts/tests/test_register.py]: There are 2 lines of the docstring.

**Review by Kuea666 — APPROVED** [review]
Coverage     88.47%  good

**Review by nicenicegame — COMMENTED** [review]
Please check this out.

File: accounts/tests/test_register.py (hunk 1 above)
Comment (ttxking) [reply accounts/tests/test_register.py]: I'll fix it now.

**Review by nicenicegame — APPROVED** [review]
Okay, Good job.

**Pull request merged**